tests/aws_iot_secure_tunneling_client_test.c

```diff
@@ -98,6 +105,13 @@ int main(int argc, char **argv) {
     ASSERT_SUCCESS(aws_condition_variable_wait(&condition_variable, &mutex));
     aws_mutex_unlock(&mutex);
 
+    char *payload = "Hi! I'm Paul / Some random payload";
+    struct aws_byte_buf buffer = aws_byte_buf_from_c_str(payload);
+    secure_tunnel->vtable.send_data(secure_tunnel, &buffer);
+    aws_mutex_lock(&mutex);
+    ASSERT_SUCCESS(aws_condition_variable_wait(&condition_variable, &mutex));
+    aws_mutex_unlock(&mutex);
```

Comment:
Could we put this in a loop and run for say 2 minutes? It would be easier to setup the source and destination together to do an manual end-to-end test.

Reply:
We could, definitely come back to that when we do the test